tests/models/test_hooks.py

```diff
@@ -611,16 +613,15 @@ def test_trainer_model_hook_system_fit_no_val_and_resume(tmpdir):
     trainer.fit(model, ckpt_path=best_model_path)
     loaded_ckpt = {
         "callbacks": ANY,
-        "epoch": 1,  # TODO: wrong saved epoch, should be 0
+        "epoch": 0,
         "global_step": 1,
         "lr_schedulers": ANY,
         "optimizer_states": ANY,
         "pytorch-lightning_version": __version__,
         "state_dict": ANY,
         "loops": ANY,
     }
-    # TODO: wrong saved epoch, should be 0
-    saved_ckpt = {**loaded_ckpt, "global_step": steps_after_reload, "epoch": 2}
+    saved_ckpt = {**loaded_ckpt, "global_step": steps_after_reload, "epoch": 1}
```

Comment:
Epoch is still 1 because the test specifies `max_steps=1` which counts as 1 epoch run (of 1 step).